src/output/icons.rs

```diff
@@ -414,6 +419,7 @@ const EXTENSION_ICONS: Map<&'static str, char> = phf_map! {
     "ejs"            => '\u{e618}',              // 
     "el"             => Icons::EMACS,            // 
     "elc"            => Icons::EMACS,            // 
+    "elf"            => Icons::BINARY,           // 
```

Comment:
Add to filetypes.rs as a FileType::Compiled

Reply:
All done